Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -137,6 +137,8 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
     WebCore::IntSize documentSize() const;
     WebCore::IntSize contentsSize() const override;
     unsigned firstPageHeight() const override;
+    unsigned pageHeight(PDFDocumentLayout::PageIndex) const;
```

Comment:
I like to call things like this `heightForPage()` which sounds less like something that would apply to all pages.